tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/robot_config.h

```diff
@@ -128,6 +128,22 @@ inline RobotConfig getRobotConfig(const tesseract_kinematics::ForwardKinematics:
 
   return RobotConfig::NDB;
 }
+
+template <typename FloatType>
+inline Eigen::Vector3i
+getRobotRedundancy(const Eigen::Ref<const Eigen::Matrix<FloatType, Eigen::Dynamic, 1>>& joint_values,
```

Comment:
I'm not sure who is owning this code, but I'd request doxygen. This looks a bit robot specific. Why are Vector3i and Vector2i for instance? I would have expected the function to be 
```
inline  Eigen::Matrix<FloatType, Eigen::Dynamic, 1>
getRobotRedundancy(const Eigen::Ref<const Eigen::Matrix<FloatType, Eigen::Dynamic, 1>>& joint_values,
const Eigen::Ref<const Eigen::Matrix<FloatType, Eigen::Dynamic, 1>>& sign_correction = ...)
```

Reply:
I agree need documentation. This is to go with the robotConfig which is for a 6DOF industrial manipulator. We will get the documentation added explaining the different inputs and outputs.

Reply:
It just seems odd that it only outputs joints 4,5,6 and the sign correction is a vector2i but only the second term is used. I'd expect a sign correction for each joint and an output for each joint. They could be Vector6i if it is only for 6 dof. However you should be able to vectorized this with Eigen for any number of joints

Reply:
@colin-lewis-19 Can you add API documentation?

Reply:
[This](https://www.onerobotics.com/posts/2013/the-perils-of-six-axis-robots/) is a good read. There are only 3, one for each config variable where that configuration can be defined multiple redundant positions. Though not sure why the return variable are labeled 4, 5 and 6 which is miss leading as they are not related to joint 4, 5 and 6 directly.

Reply:
I've added documentation and clarified naming from joints -> redundancies. I also fixed a misunderstanding on my part and now check joint 1 for redundancy 1, instead of joint 4, based on the article linked above.

However, this PR was opened by David and I am unable to push my changes to his fork.